src/api/dimensions.js

```diff
@@ -4,5 +4,9 @@ import { onError } from './index';
 // Get dimensions on startup
 export const apiFetchDimensions = () =>
     getInstance()
-        .then(d2 => d2.Api.getApi().get('dimensions.json', { paging: false }))
+        .then(d2 =>
+            d2.Api.getApi().get('dimensions.json', {
```

Comment:
Is it possible to fetch dimensions with the d2 Models api instead?  (e.g, something like d2.models.list('dimensions').

Reply:
The syntax is something like: `const dimensions = d2.models.dimensions.list();` which returns a ModelCollection object.